### DISCUSSION TITLE
TV replay stops working after sometime

### DISCUSSION THREAD

**Discussion by DROEFTOETERX** [fabianishere/udm-iptv#4] [Q&A]
Hey

First of all I'm really happy with this. It works perfectly except for one issue I have:
Regular live tv always works perfectly. However, sometimes watching tv programs back that have been on live television before (terugkijken) and watching recorded programs stops working on all 3 tv decoders. I get the error in the attached image. Live tv still works though. When I run the script again, everything will work perfectly again for around 1-3 days. I'm guessing it restarts the IGMP which solves the problem? Is there a way i can stop this from happening?

ISP is KPN so I used provided settings.
Im on v1.11.0-14 without custom kernel and I had the same issue on v1.10 with custom kern
![IMG_8404](https://user-images.githubusercontent.com/93485628/139594222-54f8889f-5fd9-4498-8d59-bbc28a0c31d3.jpg)
el

**Comment by fabianishere** [comment]
I have heard of this issue before, but haven't found a solution for this yet. 

Is there anything in the logs that might indicate an issue?

**Comment by DROEFTOETERX** [comment]
I have provided a ss from the bottom of the log, which was taken as soon as a replay stopped working. 
MRT_DEL_MFC; Errno(2): No such file or directory
Im not sure what this would mean, but this also seems to appear in my logs when its working, so I cant find anything unusual
![1](https://user-images.githubusercontent.com/93485628/139930021-59a91d98-bc52-4834-92ea-dee168f5313c.png)

**Reply by DROEFTOETERX** [reply]
An update on this,

The issue hasn't occurred in the past 3 months or so. Not sure why, but so far so good.
